File: esports-backend/src/main/java/com/esports/dto/AdminUserResponseDTO.java
package com.esports.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class AdminUserResponseDTO {
    private Long id;
    private String name;
    private String email;
    private String role;
    private String kycStatus;
    private Boolean active;
    private LocalDateTime createdAt;
}

File: Upkart1/DTO/AdminUserResponseDTO.cs
namespace Upkart1.DTO
{
    public class AdminUserResponseDTO
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Role { get; set; }
        public string KycStatus { get; set; }
        public bool Active { get; set; }
        public DateTime CreatedAt { get; set; }

        public AdminUserResponseDTO(long id, string name, string email, string role, 
            string kycStatus, bool active, DateTime createdAt)
        {
            Id = id;
            Name = name;
            Email = email;
            Role = role;
            KycStatus = kycStatus;
            Active = active;
            CreatedAt = createdAt;
        }
    }
}